scripts/convertConfig.ts

```diff
@@ -996,10 +1001,10 @@ function filterFeaturesNotReady(data: ConfigData) {
     data.mainFeatures['text-on-image'] = false;
 
     // Share is not done
-    data.mainFeatures['share-app-link'] = false;
+    // data.mainFeatures['share-app-link'] = false;
     data.mainFeatures['share-download-app-link'] = false;
     data.mainFeatures['share-apk-file'] = false;
-    data.mainFeatures['share-apple-app-link'] = false;
+    // data.mainFeatures['share-apple-app-link'] = false;
```

Comment:
For this section, change the comment to:

// Share only implements links to apps on stores.

Remove the commented out code.

Reply:
I fixed this